libs/bytes/bytes.go

```diff
@@ -69,3 +69,24 @@ func (bz HexBytes) Format(s fmt.State, verb rune) {
 		s.Write([]byte(fmt.Sprintf("%X", []byte(bz))))
 	}
 }
+
+func (bz HexBytes) MarshalJSON() ([]byte, error) {
```

Comment:
Can we add comments for why we need to add these 2 methods back and how they are called by the other function?

Reply:
Can we also add unit test for these 2 functions?